pkg/infinispan/cluster.go

```diff
@@ -125,7 +125,7 @@ func (c Cluster) GetSitesView(podName string) (map[string]bool, error) {
 	}
 	siteViews := cacheManagerInfo["sites_view"]
 	if siteViews == nil {
-		return nil, fmt.Errorf("getting Cross-Site view is not supported")
+		return nil, fmt.Errorf("Cross-site replication is not supported with the server image you are using.")
```

Comment:
Oh, @oraNod I missed this, not a Cross-site replication supported, getting Cross-Site view not supported. How to change this phrase?

Reply:
We should probably just say "retrieving the cross-site view is not supported"